lib/game.rb

```diff
@@ -48,6 +49,22 @@ def end_game
     end
   end
 
+  def play_again?
+    # asks if the player wants to restart the game
+    @display.output(PLAY_AGAIN)
+    answer = @display.input
+    if answer === "yes" || answer === "y"
+      restart
+    end
+  end
+
+  def restart
+    @board = Board.new
+    @active_player = @player_1
+    start
+    play
```

Comment:
What is start again? Just a welcome message?  Do you need to call start again on restart or just play?